ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml

```diff
@@ -34,6 +34,11 @@ promotion:
     name: 4.16-priv
     namespace: ocp-private
 releases:
+  4-16-nightly:
```

Comment:
Should we go ahead and set this to 4.17 nightly? Those are already exist and branching is next week.